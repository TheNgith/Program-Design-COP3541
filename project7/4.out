Enter species: Enter gender: Output file name: results.txt
