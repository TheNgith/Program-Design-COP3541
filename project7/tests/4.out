Buddy 2 0.30
Gila 2 0.30
